testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hudi/hive.properties

```diff
@@ -0,0 +1,9 @@
+connector.name=hive
```

Comment:
Is this property file used?

Reply:
I think the `hive` connector can be safely removed from this product test environment.
It would make sense to have in the context of [table redirections](https://trino.io/docs/current/connector/hive.html#table-redirection) , but this functionality is not yet available in Hudi.